chains/evm/contracts/test/onRamp/OnRamp/OnRamp._validateDestChainAddress.t.sol

```diff
@@ -78,7 +78,41 @@ contract OnRamp_validateDestChainAddress is OnRampSetup {
     assertEq(addressBytesLength, validated.length);
   }
 
-  // Reverts
+  function testFuzz_validateDestChainAddress_FastPathReturnsSame(
+    bytes memory rawAddress
+  ) public view {
+    vm.assume(rawAddress.length > 0 && rawAddress.length < 64);
+
+    uint8 addressBytesLength = uint8(rawAddress.length);
+    bytes memory validated = s_OnRampHelper.validateDestChainAddress(rawAddress, addressBytesLength);
+
+    assertEq(rawAddress, validated);
+  }
+
+  function testFuzz_validateDestChainAddress_PaddedNoOverflow(uint8 addressBytesLength, bytes32 tail) public view {
+    addressBytesLength = uint8(bound(addressBytesLength, 0, 32));
+
+    bytes memory padded = new bytes(32);
+    bytes memory tailBytes = abi.encodePacked(tail);
+    if (addressBytesLength != 0) {
+      uint256 offset = 32 - addressBytesLength;
+      for (uint256 i = 0; i < addressBytesLength; ++i) {
+        padded[offset + i] = tailBytes[offset + i];
+      }
+    }
+
+    bytes memory expected = new bytes(addressBytesLength);
+    for (uint256 i = 0; i < addressBytesLength; ++i) {
+      expected[i] = tailBytes[32 - addressBytesLength + i];
+    }
+
+    bytes memory validated = s_OnRampHelper.validateDestChainAddress(padded, addressBytesLength);
+    assertEq(validated, expected);
+  }
+
+  // ================================================================
+  // │                          Reverts                             │
+  // ================================================================
```

Comment:
nit we don't use these headers for this purpose so to keep it consistent maybe remove it. We used to do `// reverts` but I think we have not applied that consistently lately. I think just grouping them is clear enough for anyone reading them tbh.